src/QMCDrivers/QMCFixedSampleLinearOptimize.h

```diff
@@ -129,6 +129,10 @@ class QMCFixedSampleLinearOptimize : public QMCLinearOptimize, private NRCOptimi
   RealType max_relative_cost_change;
   ///max amount a parameter may change relative to current wave function weight
   RealType max_param_change;
+  /// the tolerance to cost function increases when choosing the best shift in the adaptive shift method
+  RealType cost_increase_tol;
+  /// the shift_i value that the adaptive shift method should aim for
+  RealType target_shift;
```

Comment:
Since it is targeting shift_i, put an explicit name seens better target_shift_i

Reply:
Change made and committed to my fork.